mobile-browser-based-version/server/src/logic/federated/handlers.js

```diff
@@ -50,7 +50,7 @@ const IDLE_DELAY = 1000 * 10;
  * Same as IDLE_DELAY, except longer for clients that recently connected and thus
  * are not critical nodes (i.e. training nodes).
  */
-const LONGER_IDLE_DELAY = 1000 * 60;
+const LONGER_IDLE_DELAY = 1000 * 600;
```

Comment:
@laurislopata @s314cy 

Is this only used for "clients that recently connected and thus are not critical nodes", if yes let's rename it to NEW_CLIENT_IDLE_DELAY.